### PR TITLE
Implement container server with unittest

### LINKED ISSUE #98: [SUJO-014] Run a task inside container server and send output out
Estimate = 6 hours

### PR DESCRIPTION
**Pull request by xkbt777**
Closes #98

### CODE REVIEW

**Comment by moody6520** [comment]
use "git rebase master" to merge with the master branch locally first, solve the conflict. Otherwise, we cannot merge this PR, and there may be duplicate commits.

File: felucca/common/task.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    def job_id(self):
+        return self.__job_id
+
+    @job_id.setter
+    def job_id(self, val):
+        self.__job_id = val
+
+    @property
+    def task_ids(self):
+        return self.__task_ids
```
Comment (Orankarl) [inline felucca/common/task.py:26]: Are there multiple ids for single task?

File: felucca/common/task.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    def __init__(self, task_executable_file, tool_type, command_line_input):
+        self.task_executable_file = task_executable_file
+        self.tool_type = tool_type
+        self.command_line_input = command_line_input
+        self.__job_id = None
+        self.__task_ids = None
+        self.__output = None
+        self.__log = None
+        self.__stdout = None
+
```
Comment (Orankarl) [inline felucca/common/task.py:15]: Do we need to add stderr here?

**Comment by xkbt777** [comment]
I'm closing this pull request because of conflict.

**Pull request closed without merging**